### ISSUE TITLE
[FEAT]: Create stand-alone web app

### ISSUE THREAD

**Issue by jakkan** [st3v3nmw/obsidian-spaced-repetition#1102]
### Is your feature request related to a problem? Please describe.

I use a different application for working with markdown than Obsidian. I still want to use spaced repetition based on my markdown files without needing to import into anki.

### Describe the solution you'd like

It would be really nice if you created a stand-alone web app for spaced repetition that could be self-hosted.

### Additional context

None

**Comment by ronzulu** [comment]
Nice idea, but this sounds more like a new product rather than a new feature of this obsidian plug-in.

This plug-in is implemented based on the obsidian APIs, and as obsidian is not available with the architecture you are after, then personally I would say this is not possible.

Cheers
Ronny

**Comment by st3v3nmw** [comment]
I agree with @ronzulu, this is not possible within Obsidian. Closing...
